wireguard-install.sh

```diff
@@ -1,5 +1,64 @@
 #!/bin/bash
 
+function addClient () {
+    # Load params
+    source /etc/wireguard/params
+
+    if [[ $SERVER_PUB_IP =~ .*:.* ]]
+    then
+        echo "IPv6 Detected"
+        ENDPOINT="[$SERVER_PUB_IP]:$SERVER_PORT"
+    else
+        echo "IPv4 Detected"
+        ENDPOINT="$SERVER_PUB_IP:$SERVER_PORT"
+    fi
```

Comment:
@angristan why do we need parenthesis?

Reply:
Because that's how you regex match in a bash if...

Reply:
If anything changes how the variables are used to anchor them. For example ${SERVER_PUB_IP} etc

Reply:
I'm sorry @AndreiG6, I meant why we need `[ ]` for IPv6. Line 10

If it's for escaping the `:` char we can use `${SERVER_PUB_IP}` instead of an `if` case.

Reply:
It's not escaping anything, just poor concat/formating I suppose. Somewhere there's probably a split for ipv6/port using that format

Reply:
No split, just an echo to the config file, but Wireguard doesn't really need this `[ ]` format.
If it's safe then I will remove the `if` case in #92 

Thanks @AndreiG6